tests/src/test/java/alluxio/server/tieredstore/SpecificTierWriteIntegrationTest.java

```diff
@@ -81,6 +81,7 @@
               LOW_WATERMARK)
           .setProperty(PropertyKey.USER_BLOCK_WRITE_LOCATION_POLICY,
               LocalFirstPolicy.class.getTypeName())
+          .setProperty(PropertyKey.WORKER_REVIEWER_ENABLED, String.valueOf(false))
```

Comment:
maybe for integration tests, we should just set the setting as disabled (or noop reviewer). I think that is in `ConfigurationTestUtils`

Reply:
Added. For the tests i modified, they are not using `ConfigurationTestUtils.defaults()`, but this will surely help the integration tests which do.